roles/kubernetes-apps/persistent_volumes/openstack/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 persistent_volumes_enabled: false
+## This parameter allows one to select the corresponding cinder device type according to the actual situation of the openstack platform
```

Comment:
Please make this a simpler comment, such as:
## Specify cinder backend type to use. See https://wiki.openstack.org/wiki/Cinder-multi-backend#Volume_Type for details

Reply:
I did, please take a look.
Virgil